### DISCUSSION TITLE
Url/Address bar - option to move it back to the top of the main area

### DISCUSSION THREAD

**Discussion by vraravam** [zen-browser/desktop#3191] [General]
I switched to the latest twilight builds, and noticed that the address bar has been moved to the top of the sidebar. Though this might increase the vertical space, for me, it loses on productivity since I used to have lots of buttons from extensions like ublock, etc that are now an extra click away. Can the old ui be brought back as a user-choosable preference?
This is how the latest twilight looks:
<img width="259" alt="Screenshot 2024-11-25 at 07 27 15" src="https://github.com/user-attachments/assets/af43de3d-4ce8-43de-89a5-879153555b9d">
This is how my equivalent in FF nightly looks (i had the same extensions, and had setup the top titlebar/toolbar to be exacrtly the same in both browsers):
<img width="1760" alt="Screenshot 2024-11-25 at 07 50 47" src="https://github.com/user-attachments/assets/a77f89f7-135e-45b9-bf2c-addd00c089f3">

**Comment by vanjaz** [comment]
Try to open about:config, and search for zen.view.use-single-toolbar. Set it to true.
This worked for me.
